src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -1787,62 +1899,98 @@ abstract class DatabaseTest {
     val reportIds = mutableListOf<ReportId>()
     val speciesIds = mutableListOf<SpeciesId>()
     val subLocationIds = mutableListOf<SubLocationId>()
+    val submissionIds = mutableListOf<SubmissionId>()
     val uploadIds = mutableListOf<UploadId>()
     val userIds = mutableListOf<UserId>()
     val withdrawalIds = mutableListOf<WithdrawalId>()
 
     val accessionId
       get() = accessionIds.last()
+
```

Comment:
Did ktfmt insert these blank lines? Interesting that it suddenly decided to do that when it hadn't previously.

Reply:
Yea it seems like it did. I recently updated the IntelliJ plugin to version 1.1.0.47. Would you like me to revert it?

Reply:
Looks like the ktfmt version in our Gradle config is out of date and Renovate hasn't been catching it. Best bet is maybe to upgrade it in Gradle and run it on the code base all at once to avoid getting piecemeal diffs like this; I'll go do that now.